workers/generic-worker/diskspace_posix.go

```diff
@@ -14,7 +14,7 @@ func freeDiskSpaceBytes(dir string) (uint64, error) {
 		return 0, err
 	}
 	// Available blocks * size per block = available space in bytes
-	b := stat.Bavail * uint64(stat.Bsize)
+	b := uint64(stat.Bavail) * uint64(stat.Bsize)
```

Comment:
What is the reason for this change?

Reply:
Without this change I get this error:

`./diskspace_posix.go:17:19: invalid operation: stat.Bavail * uint64(stat.Bsize) (mismatched types int64 and uint64)`

Reply:
Fair enough, I guess this must be an inconsistency in implementation across platforms. Then this should be fine.